src/main/java/core/basesyntax/figure/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.figure;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final Random RANDOM = new Random();
+    private static final String DEF_COLOR = Color.WHITE.name();
+    private static final int DEF_RADIUS = 10;
```

Comment:
```suggestion
    private static final String DEFAULT_COLOR = Color.WHITE.name();
    private static final int DEFAULT_RADIUS = 10;
```
It's not a common contraction